modules/core/classes/GalleryDynamicAlbum.class

```diff
@@ -37,6 +37,9 @@ class GalleryDynamicAlbum extends GalleryItem {
      */
     var $_itemTypeName;
 
+    /* deprecate dynamic properties in php 8.2 */
+    var $urlParams;
```

Comment:
Neither of these variables seem needed?

Reply:
Without both variable declarations unit tests 562, 1149 and 1612 fails with the deprecation warning from php 8.2